web/cobrands/northamptonshire/assets.js

```diff
@@ -423,7 +423,7 @@ var northants_defaults = $.extend(true, {}, fixmystreet.assets.alloy_defaults, {
       var emergency_state = ncc_is_emergency_category();
 
       disable_report_form();
-      if (!emergency_state.relevant || emergency_state.body) {
+      if ((!emergency_state.relevant || emergency_state.body) && this.visibility) {
```

Comment:
Probably need a similar update to Bucks otherwise the not_found there shows a "Pick an item" message :)

Reply:
I'm not sure I follow where this is a problem. Can you give me an example?

Reply:
I guess it instantly gets removed after, doesn't it. My thought process was (on Bucks) if you switch from e.g. Bollard light not working to non-asset category, `layer_visibilitychanged` is called, which now calls asset_not_found which calls `labeled_defaults.actions.asset_not_found` which sets the message back to the default. It doesn't really need to as it then gets removed, but then that doesn't matter if it does. Hopefully might get tidied up too.